rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -92,6 +99,21 @@ def generate_py(generator_arguments_file, typesupport_impls):
             for module_, type_ in modules[module]:
                 f.write('from %s.msg._%s import %s\n' % (args['package_name'], module_, type_))
 
+    for template_file, generated_filenames in mapping_extension_msgs.items():
+        for generated_filename in generated_filenames:
+            data = {
+                'module_name': module_name, 'package_name': args['package_name'],
```

Comment:
Please do one dictionary entry per line (I had to double take to notice there were two here).